src/pages/Home/About/Product/index.tsx

```diff
@@ -10,39 +10,44 @@ import styles from "./Product.module.scss";
 const PRODUCT_CARD = [
```

Comment:
Since these are pretty long, I think it'll be good to put them in their own file in the same folder as this